apollo-backend/src/main/java/io/logz/apollo/deployment/DeploymentHandler.java

```diff
@@ -129,6 +133,9 @@ public Deployment addDeployment(int environmentId, int serviceId, int deployable
             newDeployment.setStatus(Deployment.DeploymentStatus.PENDING);
             newDeployment.setSourceVersion(sourceVersion);
             newDeployment.setDeploymentMessage(deploymentMessage);
+
+            newDeployment.setLastUpdate(currentDate);
```

Comment:
Remove new lines.

Reply:
when creating new deployment without it, the record saved in the DB with last update equals to null